internal/compiler/expression_tree.rs

```diff
@@ -40,6 +40,7 @@ pub enum BuiltinFunction {
     SetFocusItem,
     ShowPopupWindow,
     ClosePopupWindow,
+    SelectRange,
```

Comment:
The other functions on the TextInput (select-all, copy, ...)  use ItemMemberFunction.  Is there any reason not to use that? Is that because it has arguments maybe and that's not supported by the code generator right now?
IMHO it would have been better to use `ItemMemberFunction` because it would probably be simpler,  potentially supporting different function signature. But If that's not easy, then it's fine to have it like that.

Reply:
Yes it's because of the arguments. I couldn't find an easy way to use ItemMemberFunction while still supporting arguments.